lib/doorkeeper/grants_assertion.rb

```diff
@@ -30,3 +30,12 @@ class Config
       end)
   end
 end
+
+# For compatibility with Doorkeeper <= 5.5
```

Comment:
For future readers:
https://github.com/doorkeeper-gem/doorkeeper/pull/733 introduces this in 5.5 so it's really `# For compatibility with Doorkeeper >= 5.5`